components/interactive-example/element.css

```diff
@@ -273,25 +273,21 @@ mdn-ix-tab-wrapper {
       }
 
       &.unsupported {
+        --color-unsupported: light-dark(
+          var(--color-yellow-50),
+          var(--color-yellow-80)
+        );
+
         mdn-play-editor {
-          border-color: light-dark(
-            var(--color-yellow-50),
-            var(--color-yellow-80)
-          );
+          border-color: var(--color-unsupported);
         }
 
         &::after {
-          width: 1rem;
-          height: 1rem;
-
-          content: "";
+          background-color: var(--color-unsupported);
 
-          background-image: url("../icon/triangle-alert.svg");
-          background-repeat: no-repeat;
-          background-position: center center;
-          background-size: 1rem;
-
-          opacity: 1;
+          mask-image: url("../icon/triangle-alert.svg");
```

Comment:
Note: It looks like this had regressed in https://github.com/mdn/fred/commit/08a03380352cd09047fa964ac97ce1605a3b19b8.